lib/tasks/instrumentation_generator/instrumentation.thor

```diff
@@ -60,6 +56,14 @@ class Instrumentation < Thor
 
   private
 
+  def create_instrumentation_files(base_path)
+    ['chain', 'instrumentation', 'prepend'].each do |file|
```

Comment:
I'd prefer `%w[chain instrumentation prepend]` here. I wonder if there's a cop that agrees that we're disabling currently. No action needed just now.